src/flinkSql/flinkLanguageClientManager.ts

```diff
@@ -276,7 +278,13 @@ export class FlinkLanguageClientManager implements Disposable {
       logger.warn(`Could not find environment containing compute pool ${computePoolId}`);
       return null;
     } catch (error) {
-      logger.error("Error while looking up compute pool", error);
+      let msg = "Error while looking up compute pool";
+      logger.error(msg, error);
```

Comment:
Minor: we don't really need these before logError since it will also log (possibly with additional error context): 
https://github.com/confluentinc/vscode/blob/1ddf9323514b4a1bffc18a31593d066d58f05acf/src/errors.ts#L124-L125
https://github.com/confluentinc/vscode/blob/1ddf9323514b4a1bffc18a31593d066d58f05acf/src/errors.ts#L188